05/mobile.css

```diff
@@ -0,0 +1,203 @@
+@media (min-width: 10px) and (max-width: 800px){
+body{
+    max-width: 800px;
+    margin: 0;
+    box-sizing: border-box;
+}
+
+.all{
+    max-width: 800px;
+    max-height: 100vh;
+    
+}
+.header{
+    display: grid;
+    grid-template-columns: minmax(800px);
+    grid-template-rows: minmax(200px);
+    height: 99%;
+    border: solid;
+    border-color: green;  
+}
+
+.header__row{
+    display:grid;  
+    grid-template-columns: minmax(3, 150px 300px 300px);
+    grid-template-rows: minmax(200px);
+    border: solid;
+    border-color: rgb(31, 19, 207);
+    height: 80px;
+    grid-template-areas: 
+    "A B C D";
+}
+
+.header__element{
+    display: grid;
+    grid-area: B;
+    place-self: center;
+    width: 180px;
+    height: 40px;
+    background-color: yellow;
+}
+
+
+nav{
+    display: grid;
+    grid-template-columns: 1fr;
+    grid-template-rows: 1fr;
+    grid-area: D;
+}
+
+label{
+    place-self: center;
+    width: 50px;
+    height: 35px;
+    border: 0;
+    border-top: 5px solid rgb(11, 10, 10);
+    border-bottom: 3px;
+    background-color: transparent;
+    position: relative;
+}
+
+label::before, label::after{
+    content: '';
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 100%;
+    border-top: 5px solid  rgb(11, 10, 10);
+    transform: translateY(10px);
+}
+
+label::after{
+    transform: translateY(25px);
+}
+
+nav ul {
+    display: none; /* Ukrywamy menu początkowo */
+    position: absolute;
+    top: 100px;
+
+}
+
+nav ul li{
```

Comment:
Zdecydowanie polecam stylować po klasach - ten zapis powoduje, że odwołujesz się do każdego `li` nie ważne czy `nav > ul >li` czy `nav > ul > li > ul > li` - to znacznie utrudnia stylowanie :)